app/src/main/java/com/kamron/pogoiv/PokeInfoCalculator.java

```diff
@@ -29,7 +29,10 @@ public PokeInfoCalculator(String[] namesArray, int[] attackArray, int[] defenceA
     }
 
     public Pokemon get(int number) {
-        return pokedex.get(number);
+        if (number >= 0){
+            return pokedex.get(number);
+        }
+        return null;
```

Comment:
why would this ever be called with a negative number? Having a possibility of `null` being returned means the return value should always be null checked before being used which seems like a lot of code clutter to me. 

the exception thrown seems fine to me, callers should be validating that they're doing something sane IMO